server/pulp/server/controllers/content.py

```diff
@@ -96,3 +110,87 @@ def refresh_content_source(content_source_id=None):
     conduit = ContentSourcesConduit('Refresh Content Source')
     step = ContentSourcesRefreshStep(conduit, content_source_id=content_source_id)
     step.process_lifecycle()
+
+
+@celery.task(base=Task)
+def _download(plugin_id, unit_locator, url, options):
+    """
+    Download the specified content unit.
+
+    :param plugin_id: An importer ID.
```

Comment:
I assume this is the importer object id, but since we also have the `importer_type_id` it would be nice if this was very explicit